guides/common/assembly_provisioning-virtual-machines-kvm.adoc

```diff
@@ -10,6 +10,9 @@ ifdef::context[:parent-context: {context}]
 Kernel-based Virtual Machines (KVMs) use an open source virtualization daemon and API called `libvirt` running on {RHEL}.
 {ProjectName} can connect to the `libvirt` API on a KVM server, provision hosts on the hypervisor, and control certain virtualization functions.
 
+Only Virtual Machines created through {Project} can be managed.
+Virtual Machines with other than directory storage pool types are unsupported.
```

Comment:
```suggestion
Virtual machines with other than directory storage pool types are unsupported.
```
Can you help me understand this? 
VMs with external directory storage pool types? 
I can't follow :-(

Reply:
Every VM in libvirt usually have a disk (drive), these are created as volumes in storage pools. And there are several storage pool types. Libvirt comes with preconfigured "default" directory pool type and this is the only type the Ruby library Foreman uses supports. Other like LVM pools or physical pools are not supported an will cause an error.

Reply:
https://libvirt.org/storage.html

Reply:
Are we good? Anything else?

Reply:
i have updated my suggestion. i think this is better. As it stands now, it makes no sense.